agents/postgres/pgstatsstatements/pgstatsstatements_test.go

```diff
@@ -0,0 +1,328 @@
+// pmm-agent
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package pgstatsstatements
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+	"text/tabwriter"
+	"time"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/golang/protobuf/proto"
+	"github.com/percona/pmm/api/inventorypb"
+	"github.com/percona/pmm/api/qanpb"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/reform.v1"
+	"gopkg.in/reform.v1/dialects/postgresql"
+
+	"github.com/percona/pmm-agent/utils/tests"
+)
+
+func assertBucketsEqual(t *testing.T, expected, actual *qanpb.MetricsBucket) bool {
+	t.Helper()
+	return assert.Equal(t, proto.MarshalTextString(expected), proto.MarshalTextString(actual))
+}
+
+//
+//func TestPerfSchemaMakeBuckets(t *testing.T) {
```

Comment:
I don't know about you, but from my point of view, commenting sections of code are bad practice. 
I hear a lot of arguments from both sides, but I prefer don't comment sections of code and use git.
Here is the article (for example), which explains why:  https://kentcdodds.com/blog/please-dont-commit-commented-out-code 
So maybe we should avoid doing this? At least I think we should discuss this issue and agree on what way we will choose.